Assets/Scripts/MouseTracker.cs

```diff
@@ -20,5 +24,4 @@ private void FixedUpdate()
         angle = Mathf.Atan2(mousePosition.y, mousePosition.x) * Mathf.Rad2Deg;
```

Comment:
This math is black magic as far as I know but I'm dumb 😜.  One thing that's expressed a little unclearly is this mousePosition variable. It originally represents the cursor position as I would expect. But soon you overwrite it's value with the distance between the mosue position and the fire point. I'm not sure why (comment). And once you do that, the new value shouldn't really be called mouse position, because it represents something different now.